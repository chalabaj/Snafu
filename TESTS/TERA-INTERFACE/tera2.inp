basis    6-31g

charge   	0	
spinmult	1
method 		rhf
scrdir		./scratch001
gpus		  1
run 	  	gradient
precision	mixed
convthre	1e-5
scf       diis+a

fon             yes
fon_method	gaussian
fon_temperature	0.35

casci           yes
casdoublets     4
closed          6
active          5
casnumalpha     4    # Number of alpha electrons in active space
casnumbeta      3    # Number of beta electrons in active space
castargetmult   2    # target state multiplicity
directci       yes

fmsmult         2
fmsnumstates    4
end
