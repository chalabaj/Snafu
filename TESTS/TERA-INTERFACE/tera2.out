Using MPI to get atom types and coordinates instead of reading user input file. 

Terachem server available at port_name:   tag#0$description#a29.vscht.cz$port#53402$ifname#192.168.3.29$  
Will publish this port_name to 'teraport22960'. 
Waiting to accept MPI communication from MPI client. 
MPI communication accepted.  
Reading non xyz job parameters from tera2.inp.  
Receiving initial QM atoms via MPI. We need these to determine whether there are d-functions.  

O O H H H H 


       ***********************************************************
       *                    TeraChem v1.93                       *
       *                 Hg Version: 95e7944ca4e4+               *
       *                   Development Version                   *
       *                Compiled without textures                *
       *           Chemistry at the Speed of Graphics!           *
       ***********************************************************
       * This program may only be used in connection with        *
       * a valid license from PetaChem, LLC. Use of this program *
       * or results thereof indicates acceptance of all terms    *
       * and conditions stated in the license and that a valid   *
       * license agreement between the user and PetaChem, LLC    *
       * exists. PetaChem, LLC does not warrant the correctness  *
       * of results or their suitability for any purpose.        *
       * Please email bugs, suggestions, and comments to         *
       *                  help@petachem.com                      *
       *                                                         *
       ***********************************************************


       ***********************************************************
       *  Compiled by hollas       Tue Oct 24 13:00:29 CEST 2017  *
       *  Supported architectures: sm_20 sm_30 sm_35 sm_50       *
       *  Cuda compilation tools, release 7.5, V7.5.17           *
       ***********************************************************


 Job started   Fri Mar 29 14:05:24 2019
 On a29 (available memory: 24168 MB)

######################################### RUNTIME INFO ##########################################
/usr/local/programs/custom/terachem/terachem-1.9dev/build/terachem --inputfile=tera2.inp --UseMPI=2 --MPIPort=teraport22960 

NVRM version: NVIDIA UNIX x86_64 Kernel Module  352.79  Wed Jan 13 16:17:53 PST 2016
GCC version:  gcc version 4.4.5 (Debian 4.4.5-8) 

	linux-vdso.so.1 =>  (0x00007fffe21f6000)
	libcurl.so.4 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcurl.so.4 (0x00002ba97e956000)
	libOpenMM.so => /usr/local/programs/custom/anaconda/anaconda-4.3.0/arch/x86_64/anaconda3/pkgs/openmm-7.1.1-py36_0/lib/libOpenMM.so (0x00002ba97ebb4000)
	libintbox.so.1 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libintbox.so.1 (0x00002ba97f0d6000)
	libthcbox.so.1 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libthcbox.so.1 (0x00002ba99d863000)
	libgridbox.so.1 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libgridbox.so.1 (0x00002ba99f42f000)
	libdftbox.so.1 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libdftbox.so.1 (0x00002ba99f90c000)
	libcuda.so.1 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcuda.so.1 (0x00002ba99fbee000)
	libcudart.so.7.5 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcudart.so.7.5 (0x00002ba9a0ba3000)
	libcublas.so.7.5 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcublas.so.7.5 (0x00002ba9a0e01000)
	libcufft.so.7.5 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcufft.so.7.5 (0x00002ba9a26e0000)
	libcrypto.so.1.0.0 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcrypto.so.1.0.0 (0x00002ba9a931b000)
	libssl.so.1.0.0 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libssl.so.1.0.0 (0x00002ba9a96f5000)
	libm.so.6 => /lib/libm.so.6 (0x00002ba9a995f000)
	libcilkrts.so.5 => /usr/local/programs/custom/terachem/terachem-1.9dev/build/lib/libcilkrts.so.5 (0x00002ba9a9be1000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00002ba9a9e22000)
	libgcc_s.so.1 => /lib/libgcc_s.so.1 (0x00002ba9aa136000)
	libpthread.so.0 => /lib/libpthread.so.0 (0x00002ba9aa34c000)
	libc.so.6 => /lib/libc.so.6 (0x00002ba9aa568000)
	/lib64/ld-linux-x86-64.so.2 (0x00002ba97e736000)
	libdl.so.2 => /lib/libdl.so.2 (0x00002ba9aa8ca000)
	libz.so.1 => /usr/lib/libz.so.1 (0x00002ba9aaace000)
	librt.so.1 => /lib/librt.so.1 (0x00002ba9aace5000)
#################################################################################################

Cannot find license.dat file in the TeraChem installation directory /usr/local/programs/custom/terachem/terachem-1.9dev/build


TRYING THE NETWORK LICENSE...
Connecting to license server '403-a324-01' port '8080'...
Connected!
Checking your license...

**************************************************************
  Greetings, Petr Slavicek! You have 14 licenses in total
  IN USE: 9
  AVAILABLE: 5
**************************************************************

Scratch directory: ./scratch001
Random number seed: 1763735522
Molden File Output: ./scratch001/mpi_job.molden
Using basis set: 6-31g
Spin multiplicity: 1
DIIS will use up to 10 vectors.
WF convergence threshold: 1.00e-05
Using DIIS (in Converger class) to converge WF with ADIIS+DIIS as backup
ADIIS will switch to DIIS when the DIIS error drops below 1.00e-03
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: MIXED
Single/Double precision threshold: 1.00e-03
Using fractional orbital occupations
Fractional occupations determined by gaussian broadening
6 occupied orbitals will have integer occupation
5 occupied orbitals will have fractional occupation
kT is set to  0.350 a.u.
DFT Functional requested: hf
Method: Hartree-Fock 
Wavefunction: RESTRICTED
Initial guess generated by maximum overlap

********************************************
**** SINGLE POINT GRADIENT CALCULATIONS ****
********************************************
SCF:   Initial guess is taken from previous step

using 1 out of 1 CUDA devices
   Device 0:    GeForce GTX TITAN,  6143MB, CC 3.5 -- CPU THREAD 0
-------------------------------------------------------------------
Compiled with MAGMA support.  MAGMA parameters:
    Matrices larger than 5000 square will be treated with MAGMA
       (Change by setting the MagmaMinN environment variable)
    Magma will use 1 out of 1 GPUs
       (Change by setting the MagmaNGPUs environment variable)
-------------------------------------------------------------------
 CPU Memory Available: 8058.12 MegaWords 
 GPU Memory Available: 511.98 MegaWords 
 Maximum recommended basis set size: 8100 basis functions
 (limited by GPU memory)
-------------------------------------------------------------------
Unable to find atom '' in the basis set

 Job terminated: Fri Mar 29 14:05:39 2019

/usr/local/programs/custom/terachem/terachem-1.9dev/build/terachem[0x588270]
/usr/local/programs/custom/terachem/terachem-1.9dev/build/terachem[0x66ee1d]
/lib/libc.so.6(__libc_start_main+0xfd)[0x2b4ee7084c8d]
/usr/local/programs/custom/terachem/terachem-1.9dev/build/terachem[0x443d29]

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 23034 RUNNING AT a29.vscht.cz
=   EXIT CODE: 1
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
